## Hi I'm Eric üëã

* üéì I recently completed my Ph.D. in the [Bioinformatics and Computational Biology (BCB)](https://bcb.unc.edu/) program at UNC-CH.
* üî¨ I study 3D chromatin structure and transcription in human development and disease in the [Phanstiel Lab](http://phanstiel-lab.med.unc.edu/).
* üñ• I like to build software for [Bioconductor](https://www.bioconductor.org/) to analyze, visualize, and understand genomic data.

Check out my...

* üíæ [Software](https://github.com/EricSDavis?tab=repositories)
* üìö [Publications](https://orcid.org/0000-0003-4051-3217)
* üìú [Resume](https://ericscottdavis.com/img/resume.pdf)

## Featured Workshops ‚öôÔ∏è
### [Mariner: Explore the Hi-Cs](https://ericscottdavis.com/Bioc2023mariner) (_Upcoming_)
A 90-minute Bioconductor 2023 workshop exploring Hi-C data with `mariner`, `DEseq2` and `plotgardener`. In this workshop, we use `mariner` and `DEseq2` to find differential loops in a cellular differentiation dataset. Differential loops are aggregated with `mariner` functions and visualized with `plotgardener`.

### [nullranges: Generation of nullranges via bootstrapping or covariate matching](https://ericscottdavis.com/Bioc2022nullranges)
A 45-minute Bioconductor 2023 package demo of the `nullranges` package, co-led by Wancen Mu. In this demo, we use the covariate matching and block bootstrap functionality of `nullranges` to generate sets of null-hypothesis ranges.

### [Using nullranges::matchranges() with BentoBox](https://ericscottdavis.com/Bioc2021)
A 5-minute Bioconductor 2021 lightening talk demonstrating the use of `nullranges::matchRanges()` for generating covariate-matched, null-hypothesis genomic ranges and visualizing them with `plotgardener` (formerly `BentoBox`).  
[üìΩÔ∏è Recording](https://youtu.be/nGQ6Y3GmYL4?t=770)  

## Featured Publications üìö

### [Phase separation drives aberrant chromatin looping and cancer development](https://t.co/FuYhTI65U9)
[üßµ Twitter thread](https://twitter.com/dphansti/status/1407736570636124177)

<img src="https://user-images.githubusercontent.com/31807001/184515508-39e0b56b-b665-4498-b2bf-655d66083533.png" width=50% />

### [matchRanges: Generating null hypothesis genomic ranges via covariate-matched sampling](https://t.co/Ggw3RK7Wp5)
[üßµ Twitter thread](https://twitter.com/mikelove/status/1557334149459021826) | 
[üôå Poster](https://f1000research.com/posters/10-743)

<img src="https://user-images.githubusercontent.com/31807001/184515949-7dee06c0-0cbe-4e59-8f42-172339c402b4.png" width=50% />

### [3D Chromatin Structure in Chondrocytes Identifies Putative Osteoarthritis Risk Genes](https://www.biorxiv.org/content/10.1101/2022.05.16.492146v1)
[üßµ Twitter thread](https://twitter.com/dphansti/status/1529528431930527744)

<img src="https://user-images.githubusercontent.com/31807001/184516809-0c2c14de-b269-4be3-818d-93d1449bb455.png" width=50% />

## GitHub statistics üìä

[![Eric's GitHub languages](https://github-readme-stats.vercel.app/api/top-langs?username=EricSDavis&layout=compact&hide=html,CSS&langs_count=10)](https://github.com/anuraghazra/github-readme-stats)

[![Eric's GitHub stats](https://github-readme-stats.vercel.app/api?username=EricSDavis&show_icons=true&include_all_commits=true&count_private=true)](https://github.com/anuraghazra/github-readme-stats)

## [Bioconductor](https://bioconductor.org) contributions üé∂

| Package | BioC-devel | BioC-release |
|----------------:|:----------------:|:----------------:|
| [_mariner_](https://ericsdavis.github.io/mariner/) | [![](http://bioconductor.org/shields/build/devel/bioc/mariner.svg)](http://bioconductor.org/checkResults/devel/bioc-LATEST/mariner) |[![](http://bioconductor.org/shields/build/release/bioc/mariner.svg)](http://bioconductor.org/checkResults/release/bioc-LATEST/mariner) |
| [_marinerData_](https://github.com/ericsdavis/marinerData) | [![](http://bioconductor.org/shields/build/devel/data-experiment/marinerData.svg)](http://bioconductor.org/checkResults/devel/data-experiment-LATEST/marinerData) | [![](http://bioconductor.org/shields/build/release/data-experiment/marinerData.svg)](http://bioconductor.org/checkResults/release/data-experiment-LATEST/marinerData) | N/A |
| [_nullranges_](https://nullranges.github.io/nullranges/) | [![](http://bioconductor.org/shields/build/devel/bioc/nullranges.svg)](http://bioconductor.org/checkResults/devel/bioc-LATEST/nullranges) |[![](http://bioconductor.org/shields/build/release/bioc/nullranges.svg)](http://bioconductor.org/checkResults/release/bioc-LATEST/nullranges) |
| [_nullrangesData_](https://github.com/nullranges/nullrangesData) | [![](http://bioconductor.org/shields/build/devel/data-experiment/nullrangesData.svg)](http://bioconductor.org/checkResults/devel/data-experiment-LATEST/nullrangesData) | [![](http://bioconductor.org/shields/build/release/data-experiment/nullrangesData.svg)](http://bioconductor.org/checkResults/release/data-experiment-LATEST/nullrangesData) | N/A |
| [_plotgardener_](https://phanstiellab.github.io/plotgardener/) | [![](http://bioconductor.org/shields/build/devel/bioc/plotgardener.svg)](http://bioconductor.org/checkResults/devel/bioc-LATEST/plotgardener) | [![](http://bioconductor.org/shields/build/release/bioc/plotgardener.svg)](http://bioconductor.org/checkResults/release/bioc-LATEST/plotgardener) |

## Bioinformatic Pipelines üß¨
| Pipeline | Data type(s) | Latest Version |
|---------:|:-------------|:--------------:|
| [_dietJuicer_](https://github.com/EricSDavis/dietJuicer) | Hi-C, Micro-C | [1.0.0](https://github.com/EricSDavis/dietJuicer/releases/tag/1.0.0) |
| [_MicroC_](https://github.com/EricSDavis/MicroC) | Hi-C, Micro-C | TBD |
| [_bagPipes_](https://github.com/ksmetz/bagPipes) | RNA-seq, ChIP-seq, ATAC-seq, CUT&RUN | TBD |
